Jenkinsfile_nightly

```diff
@@ -7,13 +7,32 @@ properties([
   pipelineTriggers([cron('*/30 * * * *')])
 ])
 
+def combinedSecrets = [
+  'reform-scan-aat': [
+    secret('storage-account-secondary-key', 'TEST_STORAGE_ACCOUNT_KEY'),
```

Comment:
why do we need the storage keys ? We can retrieve SAS token using endpoint `/token/sscs` and upload to blob storage  ?

Reply:
Advantage is no keys needs to be loaded less config :) plus you don't have master access

Reply:
Ok, I'll raise new PR